lib/sinon/spy.js

```diff
@@ -413,6 +413,8 @@ function delegateToCalls(method, matchAny, actual, notCalled) {
     };
 }
 
+spyApi.reset = spyApi.resetHistory;
```

Comment:
This needs to be be wrapped in our deprecation helper. See `lib/sinon/util/core/deprecated.js`